api/src/main/java/org/eclipse/microprofile/config/inject/ConfigProperty.java

```diff
@@ -135,4 +137,25 @@
      */
     @Nonbinding
     String defaultValue() default UNCONFIGURED_VALUE;
+
```

Comment:
I don't know why this diff isn't flagging it up as a change but there should be a typo in the value of UNCONFIGURED_VALUE, an extra 'd'...

```
String UNCONFIGURED_VALUE="org.eclipse.microprofile.config.configproperty.unconfigureddvalue";
```

This was previously removed but then re-instated by PR #391

Reply:
revert back to the master branch.